app/routes/customer_routes.py

```diff
@@ -0,0 +1,99 @@
+from app import db
+from app.models.customer import Customer
+from app.models.rental import Rental
+from flask import Blueprint, request, jsonify
+
+customers_bp = Blueprint("customers", __name__, url_prefix="/customers")
+
+@customers_bp.route("", methods=["GET"])
+def get_all_customers():
+    customers = Customer.query.all()
+    response_body = [customer.to_dict() for customer in customers]
+    return jsonify(response_body)
```

Comment:
solely for consistency's sake between all routes, don't forget to add a status code here

```suggestion
    return jsonify(response_body), 200
```